solidity/deploy/10_set_deposit_params.ts

```diff
@@ -10,6 +10,11 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
 
   const depositTreasuryFeeDivisor = ethers.BigNumber.from("0")
 
+  // We set the deposit reveal ahead period to 8 months and two weeks, assuming
+  // 1 month = 4 weeks = 28 days. That gives 238 days which translates to
+  // 238 * 24 * 60 * 60 = 20563200 seconds
+  const depositRevealAheadPeriod = ethers.BigNumber.from("20563200")
```

Comment:
Can we assume one month has 30 days? This will give us two weeks extra (254 days)
```
(30 * 8 + 14) * 24 * 60 * 60 = 21945600
```

Reply:
See 5ef8f99c